docs/operate/reference/viam-server/manage-viam-server.md

```diff
@@ -270,7 +270,7 @@ Then, reload the service file with the following command:
 sudo systemctl daemon-reload
 ```
 
-To resume automatic update checking, delete the leading `#` character once more, and run `sudo systemctl daemon-reload` again.
+To resume automatic update checking, delete the leading `#` character in <file>/etc/systemd/system/viam-server.service</file>, then run `sudo systemctl daemon-reload` again to reload the service file.
```

Comment:
```suggestion
To resume automatic update checking, delete the leading `#` character in <file>/etc/systemd/system/viam-server.service</file>, then run `sudo systemctl daemon-reload` again.
```
[opt] In the interest of being concise, could omit this last bit since it does say the command reloads the service file just above the first instance. I *think* it doesn't need re-explanation so soon after.

Reply:
Well spotted. Why use more word when few word do trick?